tasks/rx-install.yml

```diff
@@ -2,74 +2,74 @@
  
 - name: Add Rubrik Datos IO group
   group:
-    name: "rdio"
+    name: "mosaic"
     state: present
   register: new_group
 
 - name: Add Rubrik DatosIO User
```

Comment:
Rubrik Mosaic

Reply:
Wasn't like this in my copy. I'm going to claim it's fixed.